internal/test/framework/framework.go

```diff
@@ -294,6 +295,28 @@ func (f *TestFramework) GetNode(externalIP string) (*v1.Node, error) {
 	return matchedNode, nil
 }
 
+// ApplyHybridOverlayPatch will enable the hybrid overlay on the cluster
+func (f *TestFramework) ApplyHybridOverlayPatch() error {
+	jsonPatch := []byte(`{"spec":{"defaultNetwork":{"ovnKubernetesConfig":{"hybridOverlayConfig":` +
+		`{"hybridClusterNetwork":[{"cidr":"10.132.0.0/14","hostPrefix":23}]}}}}}`)
+
+	err := f.patchClusterNetworkOperator(jsonPatch)
+	if err != nil {
+		return fmt.Errorf("could not apply patch %s to cluster network: %s", string(jsonPatch), err)
+	}
+
+	return nil
+}
+
+// patchClusterNetworkOperator patches the "network.operator cluster" resource, with the provided JSON patch
+func (f *TestFramework) patchClusterNetworkOperator(patch []byte) error {
+	return f.OSConfigClient.RESTClient().Patch(k8stypes.MergePatchType).
+		AbsPath("/apis/operator.openshift.io/v1/networks/cluster").
+		Body(patch).
+		Do().
+		Error()
```

Comment:
Did you ping #forum-sdn as to the best way to make this API call?

Reply:
I will be switching to using the operator client library, this change is being blocked by #146. Will push revisions once that merges